ocaml-lsp-server.opam.template

```diff
@@ -1,26 +1,13 @@
 build: [
   ["dune" "subst"] {pinned}
-  # A workaround for https://github.com/ocaml/dune/issues/2982
   [
     "dune"
     "build"
     "-j"
     jobs
-    "--root"
-    "."
-    "--ignore-promoted-rules"
-    "--no-config"
-    "--profile"
-    "release"
-    "%{name}%.install"
-  ]
-  # dune 2.0 doesn't promote *.install if --ignore-promoted-rules is passed,
-  # we can ask it to do it with --promote-install-files but it's dune 2.0 only
-  # so we do this manually to be compat with dune 1.0 and dune 2.0
-  [
-    "cp"
-    "%{_:build}%/_build/default/%{name}%.install"
-    "%{name}%.install"
+    "ocaml-lsp-server/src/main.exe"
+    "@install"
```

Comment:
I think we should build `ocaml-lsp-server.install` here. That should include the binary that you've written out unnecessary targets. You should also use the `--release` flag here to set the profile correctly.

Reply:
Oh I didn't know building `ocaml-lsp-server.install` as a target was an option !
Done :)